### PR TITLE
Add script to import legacy pages and translations

### PR DESCRIPTION
**Pull request by jgmize**
Add json data from existing (legacy) site
Add django_extensions and six to requirements
Run script with './manage.py runscript create_pages'

### CODE REVIEW

**Comment by jgmize** [comment]
@glogiotatidis there's still a little more work to do on this, but with the code in the latest update I've successfully converted the german translation from the legacy json to a po file: https://github.com/mozilla/masterfirefoxos-l10n/blob/master/de/LC_MESSAGES/django.po

File: docs/internationalization.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ Python and HTML files are automatically handled by django's
 into a text file be parsable by `makemessages` command. To extract the
 database strings run::
 
-  ./manage.py cron extract_database_strings
+  ./manage.py runscript db_strings
```
Comment (glogiotatidis) [inline docs/internationalization.rst]: I believe that should be `./manage.py db_strings`

Comment (glogiotatidis) [inline docs/internationalization.rst]: Nevermind, got it

**Comment by glogiotatidis** [comment]
I think we need a fixture with the generated Pages to load it on dev and prod. Direct commits to locale directory with db-strings will be overwritten by dev's empty database.

**Comment by jgmize** [comment]
With the latest updates there is a single command to import pages and translations from the legacy data:
`./manage.py runscript import_legacy_pages_and_translations`

Please test this locally, and I'll work on some acceptance tests text. I believe we should be able to run this command in dev and prod to address the empty db issue.

**Comment by jgmize** [comment]
test this please.

**Comment by jgmize** [comment]
As we discussed in the dev meeting, I'll merge this now and add acceptance tests and further refinements in future PRs.

**Pull request merged**